databricks/koalas/groupby.py

```diff
@@ -112,14 +112,35 @@ def aggregate(self, func_or_funcs, *args, **kwargs):
         1    1    2
         2    3    4
 
+        >>> aggregated = df.groupby('A').agg('min')
+        >>> aggregated  # doctest: +NORMALIZE_WHITESPACE
+             B      C
+        A
+        1    1  0.227
+        2    3 -0.562
+
+        >>> aggregated = df.groupby('A').agg(['min', 'max'])
+        >>> aggregated  # doctest: +NORMALIZE_WHITESPACE
+             B           C
+           min  max    min    max
+        A
+        1    1    2  0.227  0.362
+        2    3    4 -0.562  1.267
         """
-        if not isinstance(func_or_funcs, dict) or \
-                not all(isinstance(key, (str, tuple)) and
-                        (isinstance(value, str) or
-                         isinstance(value, list) and all(isinstance(v, str) for v in value))
-                        for key, value in func_or_funcs.items()):
-            raise ValueError("aggs must be a dict mapping from column name (string or tuple) to "
-                             "aggregate functions (string or list of strings).")
+        if not isinstance(func_or_funcs, (str, list)):
+            if not isinstance(func_or_funcs, dict) or \
+                    not all(isinstance(key, (str, tuple)) and
+                            (isinstance(value, str) or isinstance(value, list) and
+                                all(isinstance(v, str) for v in value))
+                            for key, value in func_or_funcs.items()):
+                raise ValueError("aggs must be a dict mapping from column name (string or tuple) "
+                                 "to aggregate functions (string or list of strings).")
+
+        else:
+            group_keyname = [key.name for key in self._groupkeys]
+            agg_cols = [key for key in self._kdf.columns if key not in group_keyname]
```

Comment:
I guess `self._agg_columns` should work?

Reply:
Nice one! thanks! @ueshin